packages/retail-ui-extensions/src/extension-api/cart-api/cart-api.ts

```diff
@@ -49,9 +49,19 @@ export interface CartApiContent {
     amount?: string,
   ): Promise<void>;
 
+  /** Add a code discount to the cart
+   * @param code the code for the discount to add to the cart
+   */
+  addCartCodeDiscount(code: string): Promise<void>;
```

Comment:
Will this remain a string? Will we be moving to `CartDiscountType` in unified?

https://github.com/Shopify/ui-extensions/blob/46ce096430f5b42730d49c6ef2d9d2bbd439528b/packages/ui-extensions/src/surfaces/point-of-sale/api/cart-api/cart-api.ts#L29